### PR TITLE
API for cohorts

### PR DESCRIPTION
**Pull request by JonRojas**
Adresses (hopefully!) Tim Foley's issue [#486](https://github.com/ga-dc/garnet/issues/486)

Added to cohorts controller and form:
- Core ID
- Duration in weeks
- Day of week

Also updated the styling for the form.

I hope I did not 🔥🔥🔥🔥 anything down!

### CODE REVIEW

File: app/controllers/api/cohorts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module Api
+  class CohortsController < ApplicationController
+    def show
+      @cohort = Cohort.find(params[:id])
```
Comment (jshawl) [inline app/controllers/api/cohorts_controller.rb]: use can can can

File: app/views/cohorts/_form.html.erb [hunk 2]
```diff
@@ -5,6 +5,9 @@
     <%= f.input :name, label: false, placeholder: "Name"  %>
     <%= f.association :location, label: false, include_blank: "Location"  %>
     <%= f.association :course, label: false, include_blank: "Course" %>
+    <%= f.input :core_id, label: false, placeholder: "Core ID"  %>
+    <%= f.input :duration_in_weeks, label: false, placeholder: "Duration in Weeks"  %>
+    <%= f.input :day_of_week, as: :select, collection: [["Monday", 1], ["Tuesday", 2], ["Wednesday", 3], ["Thursday", 4], ["Friday", 5], ["Saturday", 6], ["Sunday", 7]]     %>
```
Comment (jshawl) [inline app/views/cohorts/_form.html.erb]: how to account for multiples? e.g. every monday and wednesday?

**Comment by jshawl** [comment]
:+1:

**Comment by nolds9** [comment]
:+1:

**Pull request merged**